website/docs/error-reference.md

```diff
@@ -912,9 +912,7 @@ The 4 possible values are:
 > [go/layers](http://go/layers) for more.
 
 All packages have a `layer`, which is used when checking for layering
-violations.
-
-<!-- TODO(neil): explain this further once we implement these checks -->
+violations. See [3726](#3726).
```

Comment:
I understand why we've done this but just want to say:

In an ideal world, we don't have canonical information in the error-reference.

So ideally, we would have a place for docs about the packager, and we would point people to those docs which would include a discussion of what a valid or invalid layer looks like and how to configure them, instead of the docs for another error that they didn't have.

Just something to keep in mind so that we remember to get to that eventually.

Might even be worth a TODO comment here.